### ISSUE TITLE
Manually change the order of the folders and files that are listed in VSCode File Explorer

### ISSUE THREAD

**Issue by hegedis-andor** [microsoft/vscode#95239]
Yes, this issue is a duplicate of this: https://github.com/microsoft/vscode/issues/91843
which was closed. I still wish this feature to be added. It would really make finding files easier, especially when there are a lot in the same folder. It could also follow the logical / user flow.

As a user i would like to sort my folder and files order in the explorer manually within it's parent folder, so i can decide how folders and files follow each other.

Currently it is sorted by alphanumeric, which does not follow the logicality of the project. It would make much easier to find files and understand project by looking on the logically sorted order.

Suggested implementation: drag + drop, keycombination + mouse scroll wheel.

![image](https://user-images.githubusercontent.com/22738075/79244572-57f36b80-7e77-11ea-8f64-24b59dff6335.png)

**Comment by isidorn** [comment]
This is a fair feature request, however we do not have it on our immediate plan. Thus assigning to backlog and we might add this in the future. Thanks

**Comment by hegedis-andor** [comment]
That's alright it's not on an immediate plan. I hope one day it comes thorugh. Thanks

**Comment by abhijit-hota** [comment]
+1 to this but my use case is more innocent. Now, I have a folder structure that looks like this:
```
/features
/utils
.eslintrc
.gitignore
.prettierrc
index.js
.env (gitignored)
README.md
package.json
config.json (gitignored)
package-lock.json
```

Imagine if it was like this:
```
/features
/utils
index.js
README.md
package.json
package-lock.json
.gitignore
.eslintrc
.prettierrc
.env (gitignored)
config.json (gitignored)
```
Imo, this:
Looks better. Has a better 'hierarchy' of files. Is 10 times easier to navigate.

Maybe not a feature, but someone needs to make an extension! :D

**Comment by OmranK** [comment]
Please add manual file re-ordering feature to VSCode. This is the main reason I am considering moving away from VSCode. It is extremely annoying to have files in the wrong order and not be able to reorganize them. If I'm creating a website and I want NavBar component on top since it is the first component of the website, then its not possible. Makes working in VSCode frustrating.

**Comment by david-fong** [comment]
See #119207, which is closed in favour of #27286
